### ISSUE TITLE
Json Templates

### ISSUE THREAD

**Issue by philipzaengle** [putyourlightson/craft-blitz#243]
Sometimes I need to do stupid stuff, like use twig to build objects and then return those via a template. 

A normal template would look something like this:

```
{%- spaceless -%}
{% header "Content-Type: application/json; charset=utf-8" %}
{% header "Access-Control-Allow-Origin: *" %}
{% set json = {
  status: entry ? 200 : 404,
 ... other data
} %}

{{  json | json_encode | raw }}
{%-  endspaceless -%}
```

This works super well, except when Blitz adds a caching comment to the end of the file. That breaks the JSON.

Would it be possible to either

1) Add a config option `craft.blitz.options` for enabling or disable comments on a per template basis?
2) Detect the content type and only add it to `text/html` content types?
3) Only add comments to templates then end in `html` or `twig`?

The current solution is to disable the comment timestamps all together, which is not always desired.

**Comment by bencroker** [comment]
Have you tried naming the template `stuff.json`? That should still render the template in Twig, but will also prevent Blitz from caching it. If caching the template is important to you then it sounds like a [page specific option](https://putyourlightson.com/plugins/blitz#page-specific-options) for disabling comments would solve it, which I could consider adding.

**Comment by philipzaengle** [comment]
yes, I tried changing the extension to `.json` but that didn't change the behavior. It may be because we're not calling the template with the .json extension (we're mapping it via a route). 

If you end up adding a page specific option that would be rad. Thanks for the consideration!

**Comment by bencroker** [comment]
Added `outputComments` as a page specific option in https://github.com/putyourlightson/craft-blitz/commit/5b4d5e687c3e7a4d21cf1db18398338fd6572977 for the next release.

**Comment by bencroker** [comment]
Released in version 3.6.8.
